demos/grouped_mask_prostate_longitudinal/README.md

```diff
@@ -59,14 +64,14 @@ deepreg_predict --gpu "" --config_path demos/grouped_mask_prostate_longitudinal/
 A pre-trained model will be downloaded after running [demo_data.py](./demo_data.py) and
 unzipped at dataset folder under the demo folder. This pre-trained model will be used by
 default with `deepreg_predict`. Run the user-trained model by specify `--ckpt_path` to
-where the ckpt files are save, in this case (specified by `deepreg_train` as above),
+where the ckpt files are saved, in this case (specified by `deepreg_train` as above),
 /logs/grouped_mask_prostate_longitudinal/.
 
-## Data
-
-This is a demo without real clinical data due to regulatory restrictions. The MR and
-ultrasound images used are simulated dummy images.
-
 ## Tested DeepReg Version
 
 Last commit at which demo was tested: 3157f880eb99ce10fc3a4a8ebcc595bd67be24e1
+
+## Contact
+
+Please raise an issue following the guidelines
```

Comment:
Please [raise an issue](https://github.com/DeepRegNet/DeepReg/issues/new) following the
[guidelines](../contributing/issue.html).

Sorry maybe i gave a wrong link before, please change to this